### ISSUE TITLE
 Hello Can you add a table to the minimum price of orders?

### ISSUE THREAD

**Issue by ghost** [auwaho/steam-order-scanner#2]
Is it possible to add a table  min order price to the extension.

**Comment by auwaho** [comment]
Hello, please describe in detail what exactly you mean.

**Comment by user13666** [comment]
[
![111111111111111111111111111](https://user-images.githubusercontent.com/64257330/85045771-1db88a00-b198-11ea-851d-bf865591d161.png)
](url)

**Comment by ghost** [comment]
![111111111111111111111111111](https://user-images.githubusercontent.com/64257330/85045937-5eb09e80-b198-11ea-900a-476cc14fad48.png)

**Comment by auwaho** [comment]
Yes, its possible. I will consider ability to add this function, it's pretty useful btw.

**Comment by user13666** [comment]
Здравствуйте.
У меня возникла ошибка при скане. С чем это может быть связано?
Остановилось на лоте https://steamcommunity.com/market/listings/753/785260-Silver%20%28Foil%29

scanner.js:185 Uncaught (in promise) TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at checkOrders (file:///C:/Users/123/Downloads/SORS%20(/market/2)/site/scanner.js:185:90)
checkOrders @ scanner.js:185
async function (async)
checkOrders @ scanner.js:161
setTimeout (async)
(anonymous) @ scanner.js:117
4jquery-1.11.1.min.js?v=.isFTSRckeNhC:4 GET https://steamcommunity.com/actions/GetNotificationCounts net::ERR_NAME_NOT_RESOLVED
send @ jquery-1.11.1.min.js?v=.isFTSRckeNhC:4
ajax @ jquery-1.11.1.min.js?v=.isFTSRckeNhC:4
UpdateNotificationCounts @ shared_global.js?v=PEiLJ6UP6RU9&l=russian:3308
fnDoPoll @ shared_global.js?v=PEiLJ6UP6RU9&l=russian:3265
(anonymous) @ shared_global.js?v=PEiLJ6UP6RU9&l=russian:3282
dispatch @ jquery-1.11.1.min.js?v=.isFTSRckeNhC:3
r.handle @ jquery-1.11.1.min.js?v=.isFTSRckeNhC:3
market.js?v=aOB7dZTh0EKY&l=russian:1932 WebSocket closed. Retry in 114806ms.

![Снимок](https://user-images.githubusercontent.com/64257330/85873994-9cd24180-b7da-11ea-80ee-aaecd2c728af.PNG)
